intern.json

```diff
@@ -0,0 +1,39 @@
+{
+  "coverage": false,
+  "tunnel": "browserstack",
+  "defaultTimeout": 90000,
+
+  "tunnelOptions": {
+    "verbose": true,
+    "anyOf": [
+      { "type": "string", "username": "process.env.BROWSERSTACK_USERNAME", "accessKey": "process.env.BROWSERSTACK_ACCESS_KEY" },
+      { "type": "string", "username": "BROWSERSTACK_USERNAME",  "accessKey": "BROWSERSTACK_ACCESS_KEY"}
+    ]
+  },
+
+  "capabilities": {
+    "build": "browserstack build",
+    "browserstack.source": "intern:sample-v4:v1.0",
```

Comment:
Why is there a mixture of w3c and non w3c caps? Please make it consitent with selenium version